### ISSUE TITLE
Priority 1 accessibility issue: Colour filter slider functionality is not operable

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6550]
**What**

There was functionality within the content that was not operable through a keyboard interface.

The slider within the ‘colours’ filter option cannot be accessed via keyboard. Keyboard reliant users would not be able to access this feature and have the same level of control as a user who can use a mouse. While screen reader users can navigate to the slider, they are unable to control it using the cursor keys.

Solution:
Ensure that all elements can be navigated via keyboard using the tab key and can be controlled using the cursor keys. For more information on accessible sliders, please refer to the W3C example of slider components.

Code ref
<img width="523" alt="Screenshot 2021-05-27 at 14 38 57" src="https://user-images.githubusercontent.com/7315126/119836007-46ddb080-bef9-11eb-8220-6ca875d62982.png">

Exmaple
<img width="523" alt="Screenshot 2021-05-27 at 14 39 29" src="https://user-images.githubusercontent.com/7315126/119836098-59f08080-bef9-11eb-93b1-c34f4a62729f.png">

**Where**

URL: https://wellcomecollection.org/images?query=train Page: Trains | Image search | Wellcome Collection
